prediction_market_agent_tooling/tools/web3_utils.py

```diff
@@ -237,3 +235,15 @@ def send_function_on_contract_tx_using_safe(
     receipt_tx = web3.eth.wait_for_transaction_receipt(tx_hash, timeout=timeout)
     check_tx_receipt(receipt_tx)
     return receipt_tx
+
+
+def send_xdai_to(
+    web3: Web3, from_address: ChecksumAddress, to_address: ChecksumAddress, value: Wei
+) -> None:
+    tx_params: TxParams = {
+        "from": from_address,
+        "to": to_address,
+        "value": value,
+    }
+    tx_hash = web3.eth.send_transaction(tx_params)
+    web3.eth.wait_for_transaction_receipt(tx_hash)
```

Comment:
```suggestion
    receipt_tx = web3.eth.wait_for_transaction_receipt(tx_hash)
    check_tx_receipt(receipt_tx)
```

Should this be here as with other transactions?

Reply:
Yep, that's the current implementation, please see https://github.com/gnosis/prediction-market-agent-tooling/blob/gabriel/local-chain-for-tests/prediction_market_agent_tooling/tools/web3_utils.py#L186

Reply:
Yes, but you are not using `send_function_on_contract_tx` in this function:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent-tooling/assets/15619339/f272f10d-8dfc-49dd-bcc1-8bba09493ec4)

Reply:
Great catch, added